src/fortran/curve.f90

```diff
@@ -24,7 +24,8 @@ module curve
        subdivide_nodes_generic, split_candidate, allocate_candidates, &
        update_candidates, projection_error, can_reduce
   public &
-       CurveData, LOCATE_MISS, LOCATE_INVALID, evaluate_curve_barycentric, &
+       CurveData, LOCATE_MISS, LOCATE_INVALID, evaluate_curve_vs, &
+       evaluate_curve_de_casteljau, evaluate_curve_barycentric, &
```

Comment:
I am torn here. I want these public so I can test them (especially testing `evaluate_curve_de_casteljau` for degree less than 55), however I don't (yet) want to commit to making them part of the public `libbezier` API.